daemon/README.md

```diff
@@ -239,6 +236,15 @@ program continues to run.
 
 When a Netdata program cannot run at all, a `FATAL` line is logged.
 
+### collector.log
+
+The `collector.log` is the `stderr` of all external plugins run by `netdata`.
```

Comment:
```suggestion
The `collector.log` is the `stderr` of all [external plugins](../collectors/plugins.d#provided-external-plugins) run by `netdata`.
```

@tkatsoulas would a relative link like this work to link out to make it easy for users to see what we mean by "external plugins"?

Reply:
Nice catch - that line should be removed because both internal and external collectors logs go to collectors.log

Reply:
updated my suggestion to maybe link to COLLECTORS.md so users can easily see what sort of stuff will go into `collector.log`. Is obvious but no harm being as explicit as possible with the link maybe?

Reply:
I believe it should be just `[colectors](/collectors/COLLECTORS.md)` (no need for `..`)

Reply:
Be so kind and add the github absolute link. This is the new approach. I am plannning to change every document  that contains relative links

Reply:
cool - done - question - when that link eventually end up rendered in learn would it then take user away from learn and into GH? or will learn somehow not do this?

Reply:
I think the link I suggested is not relative. The one with `..` is relative.

Reply:
I am adding the link. :)